Templates/Ros2FleetRobotTemplate/Template/Platform/Mac/mac_project.cmake

```diff
@@ -0,0 +1,8 @@
+# {BEGIN_LICENSE}
```

Comment:
question: Is it expected to always see `{BEGIN/END_LICENSE}` here? I thought this might be omitted for the created files but I could be wrong.

Reply:
I believe this is needed so that the template generation will include the comments in the generated project files for this template, otherwise this comment is meant to just represent the template file only.

Reply:
Ah understood! Sorry ignore my earlier comments @michalpelka, I forgot this was itself a template 🙈 🤦

Reply:
@hultonha actually, after reading your comment I had some doubts about these tags by myself :)
BTW, the 'Multiplayer` template doesn't have these tags.

Reply:
@michalpelka is this something you could follow-up with @pawelbudziszewski about? Can we sanity check this is correct? Just want to make sure before it gets merged. Thanks!

Reply:
Yes, I would expect ${BEGIN_LICENSE} here. It is mentioned in documentation of CLI tool here :
https://www.o3de.org/docs/user-guide/project-config/cli-reference/#usage-2
The file you commented, is empty in generated file, since `--keep-license-text` was not added to CLI call.